src/bacnet/basic/service/s_rd.c

```diff
@@ -52,7 +56,12 @@ uint8_t Send_Reinitialize_Device_Request(
         return 0;
     }
     /* is the device bound? */
-    status = address_get_by_device(device_id, &max_apdu, &dest);
+    status = address_get_by_device(
```

Comment:
this change is not needed if `address_segment_get_by_device()` is added to address.c module and address_get_by_device() remains unchanged